services/ego/index.js

```diff
@@ -113,27 +114,27 @@ const getScopes = async (userName, Authorization) => {
 };
 
 /**
- * @param {Array<EgoAccessKeyResponse>} keys
+ * @param { EgoAccessKeyResponse[] } keys
```

Comment:
Why wouldn't we pass a list of strings of the key names instead of the response? As written here, we couldn't reuse this service to delete (revoke) specific keys through a mutation...

Reply:
that's how it was before so I just kept it to avoid making mistakes and introducing bugs :P